client/my-sites/sharing/connections/service-action.jsx

```diff
@@ -1,89 +1,85 @@
+	};
 
-	render: function() {
+	render() {
 		let primary = false,
-			borderless = false,
 			warning = false,
-			isPending, removableConnections, label;
-
-		isPending = 'unknown' === this.props.status || this.props.isDisconnecting ||
-			this.props.isRefreshing || this.props.isConnecting;
+			label;
 
-		if ( 'connected' === this.props.status ) {
-			removableConnections = serviceConnections.getRemovableConnections( this.props.service.ID );
-		}
+		const { translate } = this.props,
+			isPending = 'unknown' === this.props.status || this.props.isDisconnecting ||
+				this.props.isRefreshing || this.props.isConnecting;
 
 		if ( 'unknown' === this.props.status ) {
-			label = this.translate( 'Loading…', { context: 'Sharing: Publicize status pending button label' } );
+			label = translate( 'Loading…', { context: 'Sharing: Publicize status pending button label' } );
 		} else if ( this.props.isDisconnecting ) {
-			label = this.translate( 'Disconnecting…', { context: 'Sharing: Publicize disconnect pending button label' } );
+			label = translate( 'Disconnecting…', { context: 'Sharing: Publicize disconnect pending button label' } );
 		} else if ( this.props.isRefreshing ) {
-			label = this.translate( 'Reconnecting…', { context: 'Sharing: Publicize reconnect pending button label' } );
+			label = translate( 'Reconnecting…', { context: 'Sharing: Publicize reconnect pending button label' } );
 			warning = true;
 		} else if ( this.props.isConnecting ) {
-			label = this.translate( 'Connecting…', { context: 'Sharing: Publicize connect pending button label' } );
-		} else if ( 'connected' === this.props.status && removableConnections.length ) {
-			if ( removableConnections.length > 1 ) {
-				label = this.translate( 'Disconnect All', { context: 'Sharing: Publicize disconnect button label' } );
+			label = translate( 'Connecting…', { context: 'Sharing: Publicize connect pending button label' } );
+		} else if ( 'connected' === this.props.status ) {
+			if ( this.props.removableConnections.length > 1 ) {
+				label = translate( 'Disconnect All', { context: 'Sharing: Publicize disconnect button label' } );
 			} else {
-				label = this.translate( 'Disconnect', { context: 'Sharing: Publicize disconnect button label' } );
+				label = translate( 'Disconnect', { context: 'Sharing: Publicize disconnect button label' } );
 			}
 		} else if ( 'reconnect' === this.props.status ) {
-			label = this.translate( 'Reconnect', { context: 'Sharing: Publicize reconnect pending button label' } );
+			label = translate( 'Reconnect', { context: 'Sharing: Publicize reconnect pending button label' } );
 			warning = true;
 		} else {
-			label = this.translate( 'Connect', { context: 'Sharing: Publicize connect pending button label' } );
+			label = translate( 'Connect', { context: 'Sharing: Publicize connect pending button label' } );
 			primary = true;
 		}
 
 		return (
 			<Button
 				primary={ primary }
-				borderless={ borderless }
+				borderless={ false }
```

Comment:
It used to use a variable that was never changed: https://github.com/Automattic/wp-calypso/pull/9188/files/6d94ba64f77f2929904cf68b766275a5ce7efbfd#diff-b2ee040194d578eaf77f388c884e6e10L43